### ISSUE TITLE
Fix inconsistencies between arguments for the different commands

### ISSUE THREAD

**Issue by peterjaap** [elgentos/regenerate-catalog-urls#58]
Especially concerning store scoping. To do a full regeneration for a website with two stores, we'll now need to do this mess;

```
bin/magento regenerate:product:url -s all

bin/magento regenerate:category:url -s storecode1
bin/magento regenerate:category:path -s storecode1

bin/magento regenerate:category:url -s storecode2
bin/magento regenerate:category:path -s storecode2

bin/magento regenerate:cms-page:url -a=1
```

**Comment by peterjaap** [comment]
```
bin/magento regenerate:category:path --help                
Description:
  Regenerate path for given categories

Usage:
  regenerate:category:path [options] [--] [<cids>...]

Arguments:
  cids                  Categories to regenerate

Options:
  -s, --store=STORE     Use the specific Store View [default: 0]
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

```
bin/magento regenerate:category:url --help 
Description:
  Regenerate url for given categories

Usage:
  regenerate:category:url [options] [--] [<cids>...]

Arguments:
  cids                  Categories to regenerate

Options:
  -s, --store=STORE     Use the specific Store View [default: 0]
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question

```

```
bin/magento regenerate:product:url --help
Description:
  Regenerate url for given products

Usage:
  regenerate:product:url [options] [--] [<pids>...]

Arguments:
  pids                  Product IDs to regenerate

Options:
  -s, --store=STORE     Regenerate for one specific store view
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

```
bin/magento regenerate:cms-page:url --help
Description:
  Regenerate url for cms pages.

Usage:
  regenerate:cms-page:url [options] [--] [<pids>...]

Arguments:
  pids                           CMS Pages to regenerate

Options:
  -s, --store[=STORE]            Regenerate for one specific store view [default: 0]
  -a, --all-stores[=ALL-STORES]  Regenerate for all stores. [default: false]
  -h, --help                     Display this help message
  -q, --quiet                    Do not output any message
  -V, --version                  Display this application version
      --ansi                     Force ANSI output
      --no-ansi                  Disable ANSI output
  -n, --no-interaction           Do not ask any interactive question
  -v|vv|vvv, --verbose           Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

**Comment by peterjaap** [comment]
Fixed in 0.3.0

https://github.com/elgentos/regenerate-catalog-urls/releases/tag/0.3.0
